src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ValidationErrors.feature

```diff
@@ -44,7 +44,7 @@ Scenario: Post an invalid body for academicWeeks missing schoolid for schoolRefe
 	Then the response code is 400
```

Comment:
Nothing to change here, but we are planning to go through the different feature files and make sure we are using the same Step definitions. For instance, in other files we have ` Then it should respond with 400`, basically we should have only one method.